src/redux/reducers/__tests__/__snapshots__/viewOptionsReducer.test.js.snap

```diff
@@ -9,7 +9,7 @@ Object {
       "expandedItems": Array [],
```

Comment:
required:
the `viewOptionsReducer.test.js` needs to be updated to reflect the changes in the reducer, your snapshots are out-of-date

![Screen Shot 2022-08-04 at 11 26 34 AM](https://user-images.githubusercontent.com/3761375/182885975-8319c89a-1aa6-4afb-a3d4-18bcda4b00d2.png)

Reply:
Updated!